### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by Movgto**
# Changelog:

The code was refactored to use ES6 modules, but the application behaves and looks exactly the same as it was before this change.

### CODE REVIEW

**Review by shahadat3669 — CHANGES_REQUESTED** [review]
Hi @Movgto , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.



## Invalid Code Review Request

You have submitted a project, that is invalid because:

- [ ] To ensure that your code adheres to the project's linting standards, it is important to have a green linter result. This means that the last commit in your pull request should have a green check mark (:white\_check\_mark:) or a summary indicating that all the required checks have passed. This helps in maintaining a clean and consistent codebase. Please double-check man\_technologist: your latest commit and verify that the linter checks have passed. If they haven't, make the necessary adjustments to meet the linting requirements. By having a green linter result, you demonstrate your commitment to producing high-quality code.

| Your | Professional |
| --- | --- |
| image | image |

- [ ] The branch name you have chosen does not provide a clear indication of the feature or action being implemented. It is important to use descriptive branch names that accurately describe the purpose of the branch. For example, instead of a generic name like "feature1," consider using names like "**create_sidebar**" or "**add_comments_section**" that convey the specific feature or action being worked on. :pray: Please rename the branch to a meaningful name. Check out this [official document](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-branches-in-your-repository/renaming-a-branch) for further help. :+1:

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by LionRouge1 — COMMENTED** [review]
Hi @Movgto, 

done

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't follow GitHub flow (Your branch name should have a feature or action name (e.g. "es6-module" or "Awesome-book-es6"). Kindly create a new branch with a semantic name representing what has been implemented in the branch and resubmit for review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md
Comment (Godedict) [inline README.md:1]: Your license link is broken... I suggest you fix that, it allows users ease of access to read your license detail. You can fix it by renaming your License MIT.md

**Review by Godedict — COMMENTED** [review]
Great job implementing your code 👍  kindly fix the suggested changes

**Pull request merged**